src/Appwrite/Auth/Auth.php

```diff
@@ -91,37 +91,6 @@ class Auth
      */
     public const MFA_RECENT_DURATION = 1800; // 30 mins
 
-    /**
-     * @var string
-     */
-    public static $cookieName = 'a_session';
-
-    /**
-     * User Unique ID.
-     *
-     * @var string
-     */
-    public static $unique = '';
-
-    /**
-     * User Secret Key.
-     *
-     * @var string
-     */
-    public static $secret = '';
-
-    /**
-     * Set Cookie Name.
-     *
-     * @param $string
-     *
-     * @return string
-     */
-    public static function setCookieName($string)
-    {
-        return self::$cookieName = $string;
-    }
```

Comment:
not sure if removing it from here is a good idea? what was the reason?

Reply:
It's static and will be shared between coroutines check [this](https://github.com/appwrite/appwrite/pull/7975#discussion_r1745942848)